Common subdirectories: date/date_api and date.new/date_api
Common subdirectories: date/date_migrate and date.new/date_migrate
Common subdirectories: date/date_popup and date.new/date_popup
Common subdirectories: date/date_repeat and date.new/date_repeat
diff -up date/date.theme date.new/date.theme
--- date/date.theme	2011-08-23 12:53:17.000000000 +0200
+++ date.new/date.theme	2011-09-19 08:06:15.648758345 +0200
@@ -12,8 +12,8 @@
 /**
  * Theme Start/End date combination in the view.
  *
- *   $entity->date_id 
- *     If set, this will show only an individual date on a field with 
+ *   $entity->date_id
+ *     If set, this will show only an individual date on a field with
  *     multiple dates. The value should be a string that contains
  *     the following values, separated with periods:
  *     - module name of the module adding the item
@@ -21,15 +21,15 @@
  *     - field name
  *     - delta value of the field to be displayed
  *     - other information the module's custom theme might need
- * 
+ *
  *     Used by the calendar module and available for other uses.
  *     example: 'date.217.field_date.3.test'
- * 
+ *
  *   $entity->date_repeat_show
  *     If true, tells the theme to show all the computed values
  *     of a repeating date. If not true or not set, only the
  *     start date and the repeat rule will be displayed.
- * 
+ *
  *   $dates['format'] - the format string used on these dates
  *   $dates['value']['local']['object'] - the local date object for the Start date
  *   $dates['value2']['local']['object'] - the local date object for the End date
@@ -62,7 +62,7 @@ function theme_date_display_combination(
   $add_rdf     = $vars['add_rdf'];
   $precision   = date_granularity_precision($field['settings']['granularity']);
 
-  $output = ''; 
+  $output = '';
 
   // If date_id is set for this field and the delta doesn't match, don't display it.
   if (!empty($entity->date_id)) {
@@ -77,12 +77,12 @@ function theme_date_display_combination(
   // Check the formatter settings to see if the repeat rule should be
   // displayed. Show it only with the first multiple value date.
   list($id) = entity_extract_ids($entity_type, $entity);
-  if (!in_array($id, $repeating_ids) && module_exists('date_repeat') 
+  if (!in_array($id, $repeating_ids) && module_exists('date_repeat')
   && !empty($item['rrule']) && $options['show_repeat_rule'] == 'show') {
     module_load_include('inc', 'date', 'date_repeat');
     $repeat_vars = array(
-      'field' => $field, 
-      'item' => $item, 
+      'field' => $field,
+      'item' => $item,
       'entity_type' => $entity_type,
       'entity' => $entity,
     );
@@ -106,14 +106,14 @@ function theme_date_display_combination(
     case 'value':
       $date1 = $dates['value']['formatted'];
       $date2 = $date1;
-      break;    
+      break;
     case 'value2':
       $date2 = $dates['value2']['formatted'];
       $date1 = $date2;
       break;
     default:
       $date1 = $dates['value']['formatted'];
-      $date2 = $dates['value2']['formatted'];  
+      $date2 = $dates['value2']['formatted'];
       break;
   }
 
@@ -144,7 +144,7 @@ function theme_date_display_combination(
   // Start and End dates match or there is no End date, display a complete single date.
   elseif ($date1 == $date2 || empty($date2)) {
     $output .= theme('date_display_single', array(
-      'date' => $date1, 
+      'date' => $date1,
       'timezone' => $timezone,
       'attributes' => $attributes,
       'rdf_mapping' => $rdf_mapping,
@@ -155,13 +155,13 @@ function theme_date_display_combination(
   // Same day, different times, don't repeat the date but show both Start and End times.
   // We can NOT do this if the replacement value is an integer instead of a time string.
   elseif ($has_time_string && $dates['value']['formatted_date'] == $dates['value2']['formatted_date']) {
-    // Replace the original time with the start/end time in the formatted start date.  
+    // Replace the original time with the start/end time in the formatted start date.
     // Make sure that parentheses or brackets wrapping the time will be retained in the
     // final result.
     $time = theme('date_display_range', array(
-      'date1' => $time1, 
+      'date1' => $time1,
       'date2' => $time2,
-      'timezone' => $timezone, 
+      'timezone' => $timezone,
       'attributes' => $attributes,
       'rdf_mapping' => $rdf_mapping,
       'add_rdf' => $add_rdf,
@@ -169,7 +169,7 @@ function theme_date_display_combination(
     ));
     $replaced = str_replace($time1, $time, $date1);
     $output .= theme('date_display_single', array(
-      'date' => $replaced, 
+      'date' => $replaced,
       'timezone' => $timezone,
       'attributes' => array(),
       'rdf_mapping' => array(),
@@ -180,9 +180,9 @@ function theme_date_display_combination(
   // Different days, display both in their entirety.
   else {
     $output .= theme('date_display_range', array(
-      'date1' => $date1, 
-      'date2' => $date2, 
-      'timezone' => $timezone, 
+      'date1' => $date1,
+      'date2' => $date2,
+      'timezone' => $timezone,
       'attributes' => $attributes,
       'rdf_mapping' => $rdf_mapping,
       'add_rdf' => $add_rdf,
@@ -295,7 +295,7 @@ function theme_date_display_interval($va
   }
 
   $time_ago_vars = array(
-    'start_date' => $dates['value']['local']['object'], 
+    'start_date' => $dates['value']['local']['object'],
     'end_date' => $dates['value2']['local']['object'],
     'interval' => $options['interval'],
   );
@@ -389,7 +389,7 @@ function theme_date_combo($vars) {
 
   // Group start/end items together in fieldset.
   $fieldset = array(
-    '#title' => t($instance['label']) . ' ' . ($element['#delta'] > 0 ? intval($element['#delta'] + 1) : ''),
+    '#title' => t($element['#title']) . ' ' . ($element['#delta'] > 0 ? intval($element['#delta'] + 1) : ''),
     '#value' => '',
     '#description' => !empty($element['#fieldset_description']) ? $element['#fieldset_description'] : '',
     '#attributes' => array(),
Common subdirectories: date/date_tools and date.new/date_tools
Common subdirectories: date/date_views and date.new/date_views
Common subdirectories: date/help and date.new/help
Common subdirectories: date/tests and date.new/tests
